theme/styles/settings/_vars.scss

```diff
@@ -11,3 +11,12 @@ $site-padding-x-desktop-double: 2.5rem;
 $vendor-avatar-height-mobile: 1.5rem;
 $vendor-avatar-height-desktop: 2.9375rem;
 $nav-height-desktop-xl: 3.6875rem;
+
+
+/** Heights */
+$nav-height: 4.625rem;
+$nav-height-md: 42.75rem;
+$nav-height-xl: 3.875rem;
+
```

Comment:
I think the heights are slightly off at times. Can adjust later :) 
![image](https://user-images.githubusercontent.com/43737723/110014537-8628a680-7ce8-11eb-8601-78b0190bd73b.png)